newIDE/app/src/Export/ExportLauncher.js

```diff
@@ -245,6 +247,21 @@ export default class ExportLauncher extends Component<Props, State> {
 
     return (
       <Column noMargin>
+        {!!exportPipeline.shouldHaveUniquePackageName &&
+          project.getPackageName().indexOf('com.example') !== -1 && (
+            <Line>
+              <DismissableAlertMessage
+                identifier="project-should-have-unique-package-name"
+                kind="warning"
+              >
+                <I18n>
+                  {({ i18n }) =>
+                    i18n._(exportPipeline.shouldHaveUniquePackageName)
+                  }
+                </I18n>
```

Comment:
I would recommend having here the two different messages, according to a flag `exportPipeline.packageNameWarningType` which can be `none` | `desktop` | `mobile`